### PR TITLE
fix(helm): Use curl to download the helm binary

### PR DESCRIPTION
**Pull request by kmala**
wget is not present in the container environment.

### CODE REVIEW

**Comment by helgi** [comment]
https://github.com/deis/workflow-e2e/blob/c7a5c459c4291a7cbbe5eaebe71e4b7eea2c3221/docker-test-integration.sh#L22-L41 good idea to keep downloading similar

**Comment by kmala** [comment]
@helgi changed..or did you mean the whole logic of latest and all?

File: scripts/helm.sh [hunk 1]
```diff
@@ -38,8 +38,41 @@ clean_cluster() {
 # install the kubernetes helm binary.
 install_helm() {
   local helm_version=${HELM_VERSION:-canary}
-  wget http://storage.googleapis.com/kubernetes-helm/helm-"${helm_version}"-linux-amd64.tar.gz
+  local url=https://storage.googleapis.com/kubernetes-helm/helm-"${helm_version}"-linux-amd64.tar.gz
+  # Download CLI, retry up to 5 times with 10 second delay between each
+	echo "Installing Helm CLI version '${helm_version}' via url '${url}'"
+	curl -f --silent --show-error -I "${url}"
```
Comment (helgi) [inline scripts/helm.sh]: I'd drop this - that's just a `HEAD` request

File: scripts/helm.sh [hunk 2]
```diff
@@ -38,8 +38,41 @@ clean_cluster() {
...
+	curl -f --silent --show-error --retry 5 --retry-delay 10 -O "${url}"
```
Comment (helgi) [inline scripts/helm.sh]: Spacing is inconsistent

**Review by helgi — CHANGES_REQUESTED** [review]

**Comment by helgi** [comment]
@kmala what you did is what I meant :) bar the 2 changes I pointed out

File: scripts/helm.sh (hunk 1 above)
Comment (kmala) [reply scripts/helm.sh]: removed.

File: scripts/helm.sh (hunk 2 above)
Comment (kmala) [reply scripts/helm.sh]: changed.

**Comment by helgi** [comment]
You are good to go then!

**Pull request merged**